oci8.go

```diff
@@ -435,6 +475,10 @@ func ParseDSN(dsnString string) (dsn *DSN, err error) {
 		dsnString = dsnString[len(prefix):]
 	}
 
+	dsnString, mode := splitRight(dsnString, " as ")
```

Comment:
please use query line `as=XXX` instead of ` as ` suffix. see oci8.go:453

Reply:
OK, changes are now available. 

I've also added a new as_sysdba example, using the as=sysdba query line with a comment line to compare with the "sqlplus as sysdba" usage:  "sqlplus sys/syspwd@tnsentry as sysdba"